src/main.py

```diff
@@ -1,16 +1,28 @@
-# This is a sample Python script.
+import unittest
+
+
+class ReturnClass:
+    def return_number(self):
+        return 897
+back = ReturnClass()
+resolution = back.return_number()
+print(f"number = {resolution}")
+
+class Test_ReturnClass(unittest.TestCase):
+    def test_return_number(self):
+        back = ReturnClass()
```

Comment:
What is this back? It mean nothing When you change to ExampleClass call this object here `example or something`